grails-app/controllers/au/org/ala/merit/ProjectController.groovy

```diff
@@ -225,7 +225,10 @@ class ProjectController {
             model.details.visible = model.details.visible && userHasViewAccess
 
             boolean reportsVisible = config.includesContent(ProgramConfig.ProjectContent.REPORTING) && userHasViewAccess
-            Map reportingTab = [label: 'Reporting', visible:reportsVisible, type:'tab', template:'projectReporting', reports:project.reports, stopBinding:true, services: config.services, scores:scores, hideDueDate:true, isAdmin:user?.isAdmin, isGrantManager:user?.isCaseManager]
+            //temp using showMeriPlanComparison flag to identify that it's using the new program, will create a new flag otherwise
+            String projectReportingTemplate = (showMeriPlanComparison) ? 'rdpProjectReporting' : 'projectReporting'
```

Comment:
Add a method to the ProgramConfig class getDeclarationTemplate().  This method should default to the SettingPageType.RLP_REPORT_DECLARATION but should try and do a SettingPageType.getFormName(declarationPageType) where declarationPage type is a new setting.

Reply:
Hi Chris, where is the value of the declarationPageType argument will come from?

Reply:
The program config